### DISCUSSION TITLE
Add all classes' confidence scores to the inference result

### DISCUSSION THREAD

**Discussion by yarinHayun** [ultralytics/yolov5#7660] [Q&A]
Is there a way that the results from the inference will also include the probability of all the classes and not just of the class with the highest probability?
I mean the Inference result of the given image will look like this -  [xyxy, confidence, predicted class, all classes confidence]
and not as in the current original form - [xyxy, confidence, predicted class, class name].

**Comment by glenn-jocher** [comment]
@yarinHayun you can set multi-label True to return all classes above threshold.

**Reply by faizch-ai** [reply]
This approach did not work for me.

**Comment by pourmand1376** [comment]
Just set `save-conf` parameter when calling `val.py`.
